mp/src/materialsystem/stdshaders/buildmomentumshaders.bat

```diff
@@ -5,14 +5,20 @@ rem ================================
 rem ==== MOD PATH CONFIGURATIONS ===
 
 rem == Set the absolute path to your mod's game directory here ==
-set GAMEDIR=%cd%\..\..\..\game\momentum
+set GAMEDIR="C:\GitHub\momentum\mp\game\momentum"
 
 rem == Set the relative or absolute path to Source SDK Base 2013 Singleplayer\bin ==
 rem == !!! THIS MUST BE CHANGED DEPENDING ON YOUR SETUP IF YOU WISH TO COMPILE. I CANT GET ENVIRONMENT VARIABLES TO WORK !!!
-set SDKBINDIR="%ProgramFiles(x86)%\Steam\steamapps\common\Source SDK Base 2013 Multiplayer\bin"
+set SDKBINDIR="C:\Games\MomentumDev\bin"
+
```

Comment:
The only problem I have with this now is people have to manually edit this to point to their own paths. `GAMEDIR` was relative, and should stay relative since people can have their github folder wherever (not just on C: drive), and we always want to use their checked out folders rather than some arbitrary hardcoded path.

SDKBINDIR is a little more problematic, because it *should* be their MomentumDev folder, but we can't assume we know where it is. I'm fine with it being hardcoded, but we should revisit setting `SDKBINDIR` in the future.

Reply:
SDKBINDIR can be done inside the sourcecode's folder / shadercompile
![image](https://user-images.githubusercontent.com/28888920/76713694-8e957380-675d-11ea-874c-a8f5fdd9549c.png)
folder contains the shadercompile binaries.

Reply:
I don't know whether a relative path will work for `GAMEDIR` but I'll try

Reply:
Alright yes